storage/pkg/config/defaults/defaultconfig.go

```diff
@@ -340,6 +340,8 @@ func DefaultConfig() *config.Config {
 					GRPCAddr:    "127.0.0.1:9150",
 					Services:    []string{"usershareprovider", "publicshareprovider"},
 				},
+				CS3ProviderAddr:                  "127.0.0.1:9215",
+				CS3ServiceUser:                   "95cb8724-03b2-11eb-a0a6-c33ef8ef53ad",
```

Comment:
Don't we need a default value for `CS3ServiceUserIdp`?

Reply:
You're right,  I thought we'd be good since it's picking up `OCIS_URL` as well, but I agree it's better to have a default here as well.